solver/pb/ops.proto

```diff
@@ -76,6 +76,8 @@ message OpMetadata {
 	map<string, string> description = 2;  
 	// TODO: add worker constraint, etc.
 	WorkerConstraint worker_constraint = 3;
+	// LocalSourceHint hints the shared key of local sources.
+	string LocalSourceHint = 4;
```

Comment:
Can we move this into `Op_Source`?
I think this metadata structure should not contain LLB op-specific stuff.

Reply:
This shouldn't be part of the digest. The other metadata like architecture, or entitlements is also somewhat op specific(mostly only will apply to exec op).

Reply:
Hm, actually. I don't think there is any harm in adding this as part of the digest. And I think even OS/Arch should be part of the digest for execop.

Reply:
Updated